### DISCUSSION TITLE
Asset Offline

### DISCUSSION THREAD

**Discussion by superdong69** [immich-app/immich#9369] [Q&A] [answered]
So I'm getting this when trying to watch any of the videos I've included in my external library.

Asset offline
This asset is offline. Immich can not access its file location. Please ensure the asset is available and then rescan the library.

I've tried rescanning entire library, that didn't fix anything. I haven't moved files etc, I can see the image of the video, but it won't play.

**Comment by alextran1502** [comment]
Check to make sure that the mount is OK and can be seen in the container, I suspect that your network mount might get unmounted

**Reply by superdong69** [reply]
When I go to external library management, I go to the "New External Library" I have setup for my user, and click "Validate" I get "All paths validated successfully."

Is there someplace I should be checking? Immich is running on Unraid docker, and the external library is provided on the same Unraid server, so there are no shares that could have disconnected.

**Reply by superdong69** [reply]
Anyone else have any ideas how to fix this?

**Reply by superdong69** [reply]
@alextran1502 
Have you any other suggestions?

**Reply by alextran1502** [reply, marked as answer]
path validation is one thing and checking if the container can actually see the mount is another thing. I suspect your mount point is having issue, so you should start there and make sure the mounts are properly mounted on your server

**Reply by superdong69** [reply]
I reorganized my folder structure and forgot to update the variable in docker. You were right. Sorry to bug you, I feel like a dunce.

**Reply by toluk-26** [reply]
im still having this issue. ive verified that the external library variable is the same and entered the console of both immich containers and verified that the folder had images in them. what else could i be missing or extra steps to ensure that it was not my error.
